src/worker/worker.ts

```diff
@@ -0,0 +1,37 @@
+import { readExcel } from "./read-wordlist";
+import { WorkerWrapper } from "@common/worker-wrapper";
+import {
+  compileModelFromLexicalModelSource,
+  WordListFromArray,
+} from "@predictive-text-studio/lexical-model-compiler";
+import { linkStorageToKmp } from "./link-storage-to-kmp";
+import Storage from "./storage";
+
+export class WorkerWrapperImpl implements WorkerWrapper {
+  private storage: Storage;
+
+  constructor(storage = new Storage()) {
+    this.storage = storage;
+  }
+
+  saveFile = async (name: string, file: File): Promise<ArrayBuffer> => {
```

Comment:
Why use this syntax? Not everything has to be an arrow function 😂 

```suggestion
  async saveFile(name: string, file: File): Promise<ArrayBuffer> {
```